### ISSUE TITLE
[Accessibility Visual Requirement] - No tooltip for Refresh component (MAS ID: 260681a)

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#72838]
**Steps to reproduce**:
Go to a dashboard with the ability to pick a refresh interval. Hover or focus over the refresh button. It is unclear what this button does when both off and with an interval selected.

**Actual Result**:
The Tooltip is not defined for RefreshPicker in any state.

**Expected Result**
 Tooltip should be defined for the RefreshPicker in its different states (matching its `aria-label`).
 
**Relevant WCAG Criteria:**
Could possibly make an argument for [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.1#labels-or-instructions)?

**'on' state**
![image](https://github.com/grafana/grafana/assets/6906380/a0f19682-d2a8-48e0-9723-e3fb544203d4)

**'off' state**
![image](https://github.com/grafana/grafana/assets/6906380/0857b677-6d3d-4072-aae1-f73e08a325af)

**Component**
`<RefreshPicker />`
